lib/practice_exercises.rb

```diff
@@ -1,13 +1,41 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  first = 0
+  second = first + 1
+  
+  while second <= list.length - 1
+    if list[second] == list[first]
+      list.delete_at(list.index(second))
```

Comment:
Since `delete_at` shifts all the subsequent elements left one index, this make your time complexity O(n<sup>2</sup>)